Nodejs/Product/Analysis/Analysis/GlobalBuilder.cs

```diff
@@ -1310,7 +1310,8 @@ class GlobalBuilder {
 
         private static IAnalysisSet DefineProperties(FunctionValue func, Node node, AnalysisUnit unit, IAnalysisSet @this, IAnalysisSet[] args) {
             // object, {propName: {desc}, ...}
-            if (args.Length >= 2) {
+            var maxMergeTypes = unit.Analyzer.Limits.MaxMergeTypes;
```

Comment:
I assume MaxMergeTypes always has some positive integer then - regardless of analysis settings?